styles/globals.scss

```diff
@@ -31,7 +31,7 @@ body {
 
 h1 {
   font-weight: 600;
-  font-size: 42px;
+  font-size: 36px;
```

Comment:
Was there a reason this change was made? 36 is awfully close to h2 and differs from the design guidelines we have in the figma

Reply:
I’d recommend local CSS changes rather than local ones if this fixes a specific case

Reply:
I forgot to change that back, I remember it was because I had thought the title of each overall section was too small .